Samples/Docker-Simple/Build.cmd

```diff
@@ -0,0 +1,51 @@
+@echo off
+
+echo [7mTesting prerequisites[0m
```

Comment:
I learned something, and looks cool with color, but at the same time, I think it would be preferable to just have PowerShell scripts (easier to maintain, easier to port to bash if needed (although PowerShell will be cross plat soon)).
Fine to ignore comment if you are doing something that would take some time to port to PowerShell